File: algorithms/js/twoSum/twoSum.js
let twoSum = function(nums, target) {
    let numsN = nums.length;
    for (let i = 0; i < numsN - 1; i++) {
        for (let j = i + 1; j < numsN; j++) {
            if (nums[i] + nums[j] === target) {
                return [i, j]
            }
        }
    }
    return []
}

File: algorithms/python/twoSum/twoSum.py
def twoSum3(nums: list[int], target: int) -> list[int]:
    numsN = len(nums)
    for i in range(numsN - 1):
        for j in range(i + 1, numsN):
            if nums[i] + nums[j] == target:
                return [i, j]
    return []


def twoSum2(nums, target):
    numsN = len(nums)
    for i in xrange(numsN - 1):
        for j in xrange(i + 1, numsN):
            if nums[i] + nums[j] == target:
                return [i, j]
    return []
